src/app.ts

```diff
@@ -35,8 +35,6 @@ const ORIGINAL_TITLE_TEMPLATE_PROMPT =
 const CATEGORY_PROMPT = 'product category:';
```

Comment:
CATEGORY_PROMPT_PART (or _COMPONENT)? Please also change for the other consts as they aren't actually prompts, rather components/parts of the resulting prompt